SWLOR.Game.Server/Service/ItemService.cs

```diff
@@ -491,14 +498,17 @@ public void OnModuleEquipItem()
                 NWPlayer player = _.GetPCItemLastEquippedBy();
                 NWItem oItem = (_.GetPCItemLastEquipped());
                 int baseItemType = oItem.BaseItemType;
+                Effect oEffect = _.EffectVisualEffect(579);
```

Comment:
This isn't an object so call it effect or eEffect if you really want to use that (annoying) style :D